ignite/services/chain/chain.go

```diff
@@ -186,11 +187,19 @@ func (c *Chain) RPCPublicAddress() (string, error) {
 		if err != nil {
 			return "", err
 		}
-		validator := conf.Validators[0]
-		servers, err := validator.GetServers()
-		if err != nil {
-			return "", err
+
+		var servers chainconfigv1.Servers
+
+		if len(conf.Validators) == 0 {
+			servers = chainconfigv1.DefaultServers()
+		} else {
+			validator := conf.Validators[0]
+			servers, err = validator.GetServers()
+			if err != nil {
+				return "", err
+			}
 		}
```

Comment:
I would move this to the `config` package into for example:

```go
func Servers(cfg *ChainConfig) (v1.Servers, error)
```

to follow the "pattern" used by `config.FaucetHost()`.

Then replace with something like:

```suggestion
		servers, err := config.Servers(conf)
		if err != nil {
			return "", err
		}
```

If so then the `config.Servers()` must be used in the `Commands()` function defined in this file (around line 452)

Reply:
I just realize this method is not used. So I propose to just remove it

If we should keep it, I actually think we should rather return an error, since a config without validator would mean a config that is not aimed to be used for local interaction with `serve` so the usage of this method doesn't bring any sense

Reply:
Oh, you're right it's not being used. It makes sense to remove it then 👍